atlp-rwanda/bravo-bn-be
# Configure ESLint#182201661

**What does this PR do?**
This PR configures ESLint in the project

**Description of Task to be completed?**

- [x] Define rules to be followed in `.eslintrc` file
- [x] Adding npm script to lint the project


**How should this be manually tested?**

- After cloning RP, run `npm i  ` Then `npm run lint`
- If you want to fix the fixable errors that were found you can run `npm run lint:fix`
- run` npm run prettier` to check if your branch adhere to predefined rules.

 **relevant pivotal tracker stories**
https://www.pivotaltracker.com/story/show/182201661
